src/test/java/com/gooddata/md/MetadataServiceAT.java

```diff
@@ -125,6 +130,21 @@ public void usedByBatch() {
         assertThat(usage2.getUsedBy(), hasSize(5));
     }
 
+    @Test(groups = "md", dependsOnGroups = "model")
+    public void getObjsByUris() throws Exception {
+        final MetadataService md = gd.getMetadataService();
+
+        final Map<String, String> uris =
+                md.identifiersToUris(project, asList(
+                        "attr.person.department",
+                        "attr.person.role",
+                        "fact.person.shoesize"
+                ));
+
+        final Collection<Obj> objects = md.getObjsByUris(project, uris.values());
+        assertThat(objects.stream().map(Obj::getUri).collect(Collectors.toSet()), equalTo(new HashSet<>(uris.values())));
```

Comment:
Mostly `is` is used instead of `equalTo`... but I don't insist on refactoring

Reply:
ok, will update